### PR TITLE
Merge Jazz Config UI changes into Multii-account feature

### PR DESCRIPTION
**Pull request by rajeevr2715**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### CODE REVIEW

File: core/jazz_admin/components/json-editor.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
+"use strict";
```
Comment (supritat) [inline core/jazz_admin/components/json-editor.js]: Still we need this?

File: core/jazz_admin/swagger/swagger.json [hunk 2]
```diff
@@ -90,7 +90,601 @@
...
+              },
+              "Access-Control-Allow-Origin": {
+                "type": "string"
+              }
+            }
+          }
+        },
+        "security": [
+          {
+            "{envPrefix}-{domain}-cognito-authorizer": []
```
Comment (supritat) [inline core/jazz_admin/swagger/swagger.json]: This should be cognito-admin-authorizer right?

File: core/jazz_admin/swagger/swagger.json [hunk 3]
```diff
@@ -90,7 +90,601 @@
...
```
Comment (supritat) [inline core/jazz_admin/swagger/swagger.json]: This should be cognito-admin-authorizer right?

File: core/jazz_admin/swagger/swagger.json [hunk 4]
```diff
@@ -90,7 +90,601 @@
...
```
Comment (supritat) [inline core/jazz_admin/swagger/swagger.json]: Same as above.

**Comment by SiniWilson** [comment]
I am not able to see the comments

**Pull request merged**